.gitignore

```diff
@@ -0,0 +1,2 @@
+
+2008_Flights_7M/dataset/.DS_Store
```

Comment:
Should probably make this just `.DS_Store` so it's applied everywhere.

Reply:
Done.